course/templates/course/flow-page.html

```diff
@@ -285,7 +285,7 @@
               {% for other_page_data in all_page_data %}
               <li>
                 <a href="{% url "relate-view_flow_page" course.identifier flow_session.id other_page_data.page_ordinal %}">
-                  {{ other_page_data.human_readable_ordinal }}: {{ other_page_data.title }}
+                  {{ other_page_data.human_readable_ordinal }}: {{ other_page_data.title|safe|striptags }}
```

Comment:
Why is it `|striptags` in some places and not in others?

Reply:
Sorry, it's my bad, that should be removed.  I should have proof read before submit.

Reply:
Updated.